model/src/main/java/com/ge/predix/acs/rest/PolicyEvaluationResult.java

```diff
@@ -52,7 +52,7 @@
                     + "    /v1/site/{site_id}/plant/asset/{asset_id}\n" + "would include:\n" + "    /site/1\n"
```

Comment:
@anubhavi25, this comment is incorrect.  That's a good opportunity to fix it.
Change "/v1/site/{site_id}/plant/asset/{asset_id}" to the following attribute uri template: "/v1/site/{site_id}/plant{attribute_uri}".  Then it will be resolved to "/asset/1"

Reply:
Good catch fixed